modules/templates/helper.go

```diff
@@ -127,6 +127,13 @@ func NewFuncMap() []template.FuncMap {
 			}
 			return sum
 		},
+		"Add64": func(a ...int64) int64 {
+			sum := int64(0)
+			for _, val := range a {
+				sum += val
+			}
+			return sum
+		},
```

Comment:
I think the `Add` above could be refactored to accept all `int` types, instead of introducing a new `Add64`

Reply:
Oh sorry for bothering. I am not sure whether the return type `int64` is fine. Because there are codes like:
* `{{$index := 0}}`, then `{{$tag := printf "%s-%d" $.comment.HashTag $index}}` and `{{$index = Add $index 1}}`
* `{{if lt (Add $i 1) (len $.Topics)}}`

Since Golang is a strict-type language, I haven't tested whether these codes could work. Maybe it also needs something like:

```go
switch a[0].type {
    case int:
         return int(sum)
....
}
```

So I updated the comment with: "ps: maybe not that easy, so just a suggestion."

Reply:
After some testing, it seems that Golang's typing rules in template are:

Suppose there is a function `Test64(int, int64) int64`  (or returns any)

* The argument types must be matched exactly. eg: only  `{{Test64 $i32 $i64}}` works, `{{Test64 $i64 $i64}}` and `{{Test64 $i32 $i32}}` are wrong. `Add(a ...any)` is fine to accept any type.
* The return type will change variable type, `{{$i32:=1}}{{$i32 = Test64}}` then `$i32` becomes `int64` because the variables are all `any` type
* The `{{if lt $i32 $i64}}` is right, `int` and `int64` are comparable.

So the code above should all work well with current approach.